tests/export/test_docx.py

```diff
@@ -24,34 +24,35 @@ def convert(path, *args, **kwargs):
 
 class ConvertDocxToHtmlTestCase(DocXFixtureTestCaseFactory):
     cases = (
-        'read_same_image_multiple_times',
         'all_configured_styles',
+        'export_from_googledocs',
+        'external_image',
+        'has_missing_image',
+        'has_missing_image',
         'has_title',
         'inline_tags',
-        'has_missing_image',
         'justification',
         'list_in_table',
-        'external_image',
-        'export_from_googledocs',
-        'has_missing_image',
         'lists_with_styles',
         'missing_numbering',
         'missing_style',
         'nested_lists',
         'nested_table_rowspan',
         'nested_tables',
         'no_break_hyphen',
+        'read_same_image_multiple_times',
+        'rotate_image',
         'shift_enter',
         'simple',
         'simple_lists',
         'simple_table',
         'special_chars',
         'styled_bolding',
         'table_col_row_span',
+        'table_with_multi_rowspan',
         'tables_in_lists',
+        'textbox',
```

Comment:
I'm not trying to say each docx file has one and only one thing it's testing. I want to make sure that newly considered nodes exist in a docx file that we are parsing in a test. This ensures that, at least when the new node is first added, that we have a real life example that we know now works. I feel pretty strongly about having each of the nodes being processed in our tests from docx files. Mostly because back when I would fake it (in XML), I created more than a few instances of XML that could not exist in an actual docx. Meaning I was writing fixes for things that could not happen. Meaning I quite likely didn't solve the problem.